front/lib/api/data_sources.ts

```diff
@@ -81,6 +81,7 @@ export async function getDataSource(
 
   return {
     id: dataSource.id,
+    createdAt: dataSource.createdAt.getTime(),
```

Comment:
Do we really need this in the FE? Can't we track those in the BE?

Reply:
I need it in the DataSourceType, which gets used in the frontend and the backend. We have it for all other types where we did not forgot to add it.